src/main/java/academy/mischok/learningjournal/model/Tag.java

```diff
@@ -0,0 +1,21 @@
+package academy.mischok.learningjournal.model;
+
+import jakarta.annotation.sql.DataSourceDefinition;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import lombok.*;
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+
+public class Tag {
+
+    @Column(name = "id")
```

Comment:
Geht diese ID so als automatischer Primary Key?

Reply:
id and GeneratedValues added